<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Gerente</class>
 <widget class="QWidget" name="Gerente">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>884</width>
    <height>596</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="menuL" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>61</width>
     <height>601</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#menuL{
	background-color: #4F7302;
}</string>
   </property>
   <widget class="QPushButton" name="btnSalir">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>530</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#btnSalir{
	background: none;
	border: none;
	image: url(:/imagenes/img/logOff.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnGestionUsr">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>50</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#btnGestionUsr{
	background: none;
	border: none;
	image: url(:/imagenes/img/gestionUsuarios.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="menuH" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>891</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#menuH{
	background-color: #4F7302;
	border: none;
}</string>
   </property>
   <widget class="QLabel" name="relojD">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>10</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;HelvLight&quot;;
color: #F29F05;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>60</y>
     <width>831</width>
     <height>541</height>
    </rect>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>10</y>
       <width>181</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 16pt &quot;HelvLight&quot;;
color: #D96704;</string>
     </property>
     <property name="text">
      <string>Gestionar usuarios</string>
     </property>
    </widget>
    <widget class="QTabWidget" name="tabWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>831</width>
       <height>491</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">tabWidget{
	color: #F29F05
	font: 12pt &quot;HelvLight&quot;;
}
QLineEdit{
	border-right: 2px solid #D96704;
	border-bottom: 2px solid #D96704;
	font: 12pt &quot;HelvLight&quot;;
}
QLineEdit:Hover{
	border-right: 2px solid #F29F05;
	border-bottom: 2px solid #F29F05;
}</string>
     </property>
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="Agregar">
      <attribute name="title">
       <string>Agregar usuarios</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="Eliminar">
      <attribute name="title">
       <string>Eliminar usuarios</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="Modificar">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <attribute name="title">
       <string>Modificar usuarios</string>
      </attribute>
      <widget class="QPushButton" name="pushButton_2">
       <property name="geometry">
        <rect>
         <x>530</x>
         <y>400</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-color: #4F7302;
	font: 12pt &quot;HelvLight&quot;;
	color: white;
}

QPushButton:Hover{
	background-color: #D96704;
}</string>
       </property>
       <property name="text">
        <string>Cancelar</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_3">
       <property name="geometry">
        <rect>
         <x>640</x>
         <y>400</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-color: #4F7302;
	font: 12pt &quot;HelvLight&quot;;
	color: white;
}

QPushButton:Hover{
	background-color: #D96704;
}</string>
       </property>
       <property name="text">
        <string>Guardar</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>450</x>
         <y>40</y>
         <width>351</width>
         <height>351</height>
        </rect>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="img.qrc">:/imagenes/img/Logo.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
       <property name="openExternalLinks">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QWidget" name="widget" native="true">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>90</y>
         <width>371</width>
         <height>341</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QWidget{
	background-color: transparent;
	border-radius: 20px;
}
QLineEdit{
	border-bottom: 2px solid #D96704;
	font: 12pt &quot;HelvLight&quot;;
}

QLineEdit:Hover{
	border-bottom: 2px solid #F29F05;
}
</string>
       </property>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>71</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>HoloLens MDL2 Assets</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Nombre:</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>40</y>
          <width>321</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit{
	border-bottom: 2px solid #D96704;
	font: 12pt &quot;HelvLight&quot;;
}

QLineEdit:Hover{
	border-bottom: 2px solid #F29F05;
}</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>90</y>
          <width>81</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>HoloLens MDL2 Assets</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Direccion:</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_4">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>200</y>
          <width>321</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_5">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>280</y>
          <width>321</width>
          <height>31</height>
         </rect>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>120</y>
          <width>321</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>170</y>
          <width>71</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>HoloLens MDL2 Assets</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Telefono:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>250</y>
          <width>101</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>HoloLens MDL2 Assets</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Contraseña:</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="widget_2" native="true">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>20</y>
         <width>351</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QWidget{
	background-color: transparent;
	border-radius: 20px;
}
QLineEdit{
	border-right: 2px solid #D96704;
	border-bottom: 2px solid #D96704;
	font: 12pt &quot;HelvLight&quot;;
}

QLineEdit:Hover{
	border-bottom: 2px solid #F29F05;
}
QPushButton{
	background-color: #4F7302;
	font: 10pt &quot;HelvLight&quot;;
	color: white;
}
QPushButton:Hover{
	background-color: #D96704;
}
</string>
       </property>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>101</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>HoloLens MDL2 Assets</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>No. usuario:</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_3">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>10</y>
          <width>71</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>10</y>
          <width>81</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Buscar</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2"/>
  </widget>
 </widget>
 <resources>
  <include location="img.qrc"/>
 </resources>
 <connections/>
</ui>
